<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_title">
          <property name="text">
           <string>Original</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_img">
          <property name="text">
           <string>Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="5" column="0">
           <widget class="QLabel" name="label_white_y">
            <property name="text">
             <string>White y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_filename">
            <property name="text">
             <string>Filename</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_cuts">
            <property name="text">
             <string>Cuts</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_white">
            <property name="text">
             <string>White x</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSlider" name="slider_n_cuts">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btn_x_cut">
            <property name="text">
             <string>X cut</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btn_y_cut">
            <property name="text">
             <string>Y cut</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_load">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="spin_white_treshold">
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSlider" name="slider_white_treshold">
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QSpinBox" name="spin_n_cuts"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txt_filename">
            <property name="text">
             <string>/tmp/ar.pdf</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="spin_page"/>
          </item>
          <item row="5" column="1">
           <widget class="QSlider" name="slider_white_y">
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="spin_white_y">
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QPushButton" name="btn_cut">
            <property name="text">
             <string>Cut</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QSpinBox" name="spin_block"/>
          </item>
          <item row="8" column="0" colspan="3">
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_ratio">
            <property name="text">
             <string>Pixel ratio</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="label_ratio_res">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spin_white_treshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_cut</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_page</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_cut</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_white_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_white_y</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_white_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_cut</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_white_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_white_y</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
